app/controllers/admins/rewards_controller.rb

```diff
@@ -0,0 +1,61 @@
+module Admins
+  class RewardsController < AdminsController
+    def index
+      @rewards = Reward.all
+    end
+
+    def show
+      @reward = Reward.find(params[:id])
+    end
+
+    def edit
+      @reward = Reward.find(params[:id])
+    end
+
+    def new
+      @reward = Reward.new
+    end
+
+    def create
+      @reward = Reward.new(reward_params)
+
+      if @reward.title.empty?
```

Comment:
this is not needed at all and in fact an overkill. Google please for Rails validations. A model should validate the object and if there is a missing title the action should render a view with an instance of the object, then in the view you should render these errors to the user.

Reply:
Thanks! Now I know more about validations and it is and will be very useful knowledge :)